WeakAuras/Animations.lua

```diff
@@ -160,6 +158,9 @@ end
 local updatingAnimations;
 local last_update = GetTime();
 local function UpdateAnimations()
+  if not updatingAnimations then
```

Comment:
I'd prefer if instead of using `updatingAnimations` the visibility state of `frame` was changed to stop running OnUpdate, but since it needs a later refactor, up to you.